distribution/docs/src/main/resources/_contents/_catalog-contents/integrating-catalog-contents.adoc

```diff
@@ -1118,7 +1144,7 @@ After creating a custom Ftplet, it needs to be added to the FTP server’s Ftple
 
 ====== From an FTP client:
 
-The FTP endpoint can be accessed from any FTP client of choice. Some common clients are FileZilla, PuTTY, or the FTP client provided in the terminal. The default port number is *8021*. Valid usernames and password are stored in the `<INSTALL_DIR>/etc/users.properties` file.
+The FTP endpoint can be accessed from any FTP client of choice. Some common clients are FileZilla, PuTTY, or the FTP client provided in the terminal. The default port number is *8021*. Valid usernames and password are stored in the `<INSTALL_DIR>/etc/users.properties` file. If FTPS is enabled with 2 way authentication, a client that supports client authorization is required.
```

Comment:
You don't need to tell them that valid usernames/passwords are in users.properties. On a hardened system, this file is removed anyways. Valid users are any users that can log in via our security framework.
Also "2 way authentication" -> "2-way TLS" and "client authorization" -> "client authentication"